diam/autogen.sh

```diff
@@ -86,6 +86,7 @@ func init() {
 	Default, _ = NewParser()
 	Default.Load(bytes.NewReader([]byte(baseXML)))
```

Comment:
It'd be better if you could open a separate issue to track this. I haven't looked deeply into details yet but can do later.